sycl/source/detail/queue_impl.hpp

```diff
@@ -230,10 +230,6 @@ class queue_impl {
     try {
       return submit_impl(CGF, Self, Self, SecondQueue, Loc, PostProcess);
     } catch (...) {
-      {
-        std::lock_guard<std::mutex> Lock(MMutex);
-        MExceptions.PushBack(std::current_exception());
-      }
```

Comment:
This behavior was incorrect. Synchronous errors during submit should not be reported asynchronously.